### PR TITLE
SA-6070-luk-button-in-expanded-controller

### PR DESCRIPTION
**Pull request by rootkc**
Should fix the problem with stop button in the expanded view. The problem is that the original stop event closes the connection. Tested on Android and iOS with chromecast version 1.23 and 1.24

### CODE REVIEW

File: player.js [hunk 1]
```diff
@@ -1440,7 +1440,7 @@ sampleplayer.CastPlayer.prototype.onStop_ = function(event) {
   sampleplayer.transition_(self.element_, sampleplayer.TRANSITION_DURATION_,
       function() {
         self.setState_(sampleplayer.State.IDLE, false);
-        self.onStopOrig_(event);
+        // self.onStopOrig_(event);
```
Comment (jaroki) [inline player.js]: I don't know how you have tested this, but this doesn't properly cleans up the session, and the apps still see the the receiver app as playing with the solution. 

Please look into using the original solution as I suggested earlier:
https://developers.google.com/cast/docs/reference/receiver/cast.receiver.MediaManager#resetMediaElement

**Comment by jjepsen** [comment]
Please also add the "Custom code" tag.

File: player.js (hunk 1 above)
Comment (jjepsen) [inline player.js]: I'm having the same issue with setState it only affects the UI state, not the playback.

Comment (rootkc) [inline player.js]: now i'm calling the resetMediaElement_() function it should work

**Pull request merged**